action.yml

```diff
@@ -230,6 +231,24 @@ runs:
         PACKAGIST_USERNAME: ${{ env.PACKAGIST_USERNAME }}
         PACKAGIST_TOKEN: ${{ env.PACKAGIST_TOKEN }}
 
+    # Publishing the package to nuget.org if the release is a C# package
+    - name: Publish to nuget.org
+      if: contains(github.event.head_commit.message, 'chore(release)') && inputs.release-type == 'csharp'
+      uses: ./.github/actions/publish-csharp-nuget
+      with:
+        package-path: ${{ inputs.package-path }}
+      env:
+        NUGET_DIR: ${{ env.NUGET_DIR }}
+        NUGET_APIKEY: ${{ env.NUGET_APIKEY }}
+
+    # Publishing the package to pub.dev if the release is a Dart package
+    - name: Publish to pub.dev
+      # if: contains(github.event.head_commit.message, 'v[0-9]+.[0-9]+.[0-9]+*') && inputs.release-type == 'dart'
+      if: contains(github.event.head_commit.message, 'chore(release)') && inputs.release-type == 'dart'
```

Comment:
U toho Dartu to budeme muset udělat jinak. pub.dev totiž vyžaduje publish jen z GIT tagu: https://dart.dev/tools/pub/automated-publishing#publishing-packages-using-github-actions

> info Note: Pub.dev only allows automated publishing from GitHub Actions when the workflow is triggered by pushing a git tag to GitHub. Pub.dev rejects publishing from GitHub Actions triggered without a tag. This ensures that new versions cannot be published by events that should never trigger publishing.

viz https://github.com/ButterCMS/buttercms-dart/pull/9/files#diff-551d1fcf87f78cc3bc18a7b332a4dc5d8773a512062df881c5aba28a6f5c48d7